cypress/integration/User Management/BL_00_ReceptionistEdit.js

```diff
@@ -0,0 +1,128 @@
+describe("Fills out form with mock data of receptionist from our user.json file, view our form to see if the data is inputed properly, edit our form with edited data, and submits our form", () => {
+    before(() => {
+        cy.fixture("users.json").then(({ receptionist }) => {
+            cy.mockGraphQL({
+                "CreateAdmin": {
+                    "response": {
+                        "data": {
+                            "createAdmin": {
+                                "created": false,
+                            },
+                        },
+                    },
+                    "test": (variables) => {
+                        Object.entries(variables).forEach(([key, value]) => {
+                            if (!["phoneNumber", "id", "user", "password"].includes(key)) {
+                                expect(receptionist[key] || receptionist?.user[key]).equals(value);
+                            }
+                        });
+                    },
+                },
+                "GetAdmin": {
+                    "response": {
+                        "data": {
+                            "admin": {
+                                ...receptionist,
+                                "password": null,
+                            },
+                        },
+                    },
+                    "test": ({ userID }) => {
+                        expect(userID).equals(receptionist.user.id.toString(), "Check ID passed");
+                    },
+                },
+            });
+            cy.visitAuthenticated(`/form/admin/${receptionist.user.id}`);
+        });
+    });
+
+    it("Loads our mock data properly into the login details form section for receptionist", () => {
```

Comment:
rename to something more high level: "Loads existing receptionist account login details into Login Details section"